src/Extensions/AzureBlobStorage/AzureBlobEgressProviderOptions.cs

```diff
@@ -55,6 +55,7 @@ internal sealed partial class AzureBlobEgressProviderOptions
         [Display(
             ResourceType = typeof(OptionsDisplayStrings),
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_CommonEgressProviderOptions_CopyBufferSize))]
+        [Range(1, int.MaxValue)]
```

Comment:
I added range validation for this to keep it aligned with S3 - but it also means we fail the egress operation if this isn't set correctly (as opposed to just disregarding it). I would be fine with removing the Range check and just internally enforcing that CopyBufferSize > 0 if people prefer.

Reply:
I think as long as we have a reasonable default here, we should leave the range check